bitrise/template_util.go

```diff
@@ -23,14 +24,25 @@ type TemplateDataModel struct {
 }
 
 func getEnv(key string, envList envmanModels.EnvsJSONListModel) string {
-	if len(envList) > 0 {
+	envfilePath := os.Getenv(envfile.DefaultEnvfilePathEnv)
```

Comment:
What about passing the `envfilePath` as a function parameter? Or even the already opened env file (probably wrapped by a helper structure)?